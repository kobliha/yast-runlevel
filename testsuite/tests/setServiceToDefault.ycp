// test for serServiceToDefault
{
    include "testsuite.ycp";
    include "runlevel/ui.ycp";

    Runlevel::services = $[
	"a" : $[
	    "start" : [ "1", "2" ],
	    "defstart" : [ "3", "4" ],
	    ]
	];
    setServiceToDefault ("a");
    DUMP (Runlevel::services);
}
