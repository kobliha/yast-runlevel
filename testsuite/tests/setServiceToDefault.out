Dump	$["a":$["defstart":["3", "4"], "dirty":true, "start":["3", "4"]]]
